packages/stream_chat_flutter/lib/src/message_widget/message_widget.dart

```diff
@@ -1009,7 +1009,7 @@ class _StreamMessageWidgetState extends State<StreamMessageWidget>
           title: Text(context.translations.copyMessageLabel),
           onClick: () {
             Navigator.of(context, rootNavigator: true).pop();
-            Clipboard.setData(ClipboardData(text: widget.message.text));
+            Clipboard.setData(ClipboardData(text: widget.message.text ?? ''));
```

Comment:
Can you instead check for the text to be non-null?

```suggestion
            final text = message.text;
            if (text != null) Clipboard.setData(ClipboardData(text: text));
```